src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -139,6 +141,16 @@ class CompanyStep extends React.Component {
         this.clearError(inputKey);
     }
 
+    /**
+     * Clear errors associated with incorporation date.
+     *
+     * @param {String} value
+     */
+    clearIncorporationDateErrorsAndSetValue(value) {
```

Comment:
`clearError` will clear multiple errors at once if you pass them. I think a better way to write this would be 

```js
clearDateErrorsAndSetValue(value) {
    clearError(['incorporationDate', 'incorporationDateFuture']);
    this.setValue(value);
}
```

Reply:
Ok got it. i.e rename `clearIncorporationDateErrorsAndSetValue()` -->  `clearDateErrorsAndSetValue()`
and function code should be as under. ( I did small syntax correction)
``` javascript
clearDateErrorsAndSetValue(value) {
    this.clearError(['incorporationDate', 'incorporationDateFuture']);
    this.setValue({incorporationDate: value});
}
```
I also tested locally with this correction. It is working as expected. I will push new commit in few hours.

Reply:
Ah, that was my bad, I was looking at the wrong method when I thought `clearError` could take multiple error keys.
Could you change it to this then? I think it's more readable and oblivious that the errors are different. 

```js
clearDateErrorsAndSetValue(value) {
        this.clearError('incorporationDate'); 
        this.clearError('incorporationDateFuture');
        this.setValue({incorporationDate: value});
}
```

Also would you change the comment on this method to: 
`Clear both errors associated with incorporation date, and set the new value.`

Reply:
Yes this looks better. Updated it as per your request also pushed new commit.